src/biome/text/modules/heads/defs.py

```diff
@@ -58,9 +58,9 @@ def as_string(self) -> str:
 class TaskHead(torch.nn.Module, Registrable):
     """Base task head class"""
 
-    def __init__(self, model: Model):
+    def __init__(self, backbone: BackboneEncoder):
         super(TaskHead, self).__init__()
-        self.model = model
+        self.backbone = backbone
```

Comment:
Here i would also be more specific: `self.backbone` -> `self.backbone_encoder`
Here i think it is important, since this avoids a "what the fuck is a backbone?" moment in all the docstrings of the heads (TextClassification, LanguageModeling, ...)

Reply:
The backbone encoder should be described at some early point in documentation. So, at this point, users should be sufficiently familiar with the concepts of the library. Isn't it?